placebo/pill.py

```diff
@@ -21,9 +21,15 @@
 
 from placebo.serializer import serialize, deserialize
 
+from base64 import b64encode
+from base64 import b64decode
+
 LOG = logging.getLogger(__name__)
 DebugFmtString = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
 
+b64_operations = {
+    'kms': ['Decrypt', 'Encrypt', 'GenerateDataKeyWithoutPlaintext', 'GenerateDataKey']
```

Comment:
Not sure if you're concerned about PEP8, but if you are, consider fixing this line.